python/usb_console/simulation.py

```diff
@@ -136,3 +138,81 @@ def stop(self, data_dir):
 
         with open(data_dir + "/simulation_log.txt", "w") as fp:
             fp.write(self.log)
+
+class SingleSimulation(Simulation):
```

Comment:
@kkrol27 this kind of abstraction is how the company we love built our most favorite, gnarly test product

Reply:
Yep looks like it haha